docs/indexing.rst

```diff
@@ -19,11 +19,29 @@ An indexer is a service with three endpoints:
 - ``POST fields`` converts the contents of new file version into a json document that matches the json mapping
 - ``GET types`` returns mimetypes that the indexer accepts, including the 'subtypes' of a mimetype, which will handled in the same way as their parent mimetype.
 
-  - For example ``application/xml`` could have as a subtype ``application/vnd.prima.page+xml``, meaning that the indexer handles page xml just like ordinary xml.
+  - See JSON schema below.
+  - For example: ``application/xml`` could have as a subtype ``application/vnd.prima.page+xml``, meaning that the indexer handles pagexml just like ordinary xml.
   - An indexer that returns no types (status ``204 No Content``) is assumed to handle all file types
 
 More on: `ES mappings <https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html>`_.
 
+JSON Schema of types endpoint: ::
```

Comment:
Gegeven dat dit handmatige documentatie is, moet dit misschien nog ergens in de code terugkomen: bijv. "if you change this object, remember to update docs in 'indexing.rst'"?

Reply:
zie b90cb76

Reply:
zie 1452a30